### ISSUE TITLE
avMNIST baseline performance upper bound can't achieve reported performance in the paper

### ISSUE THREAD

**Issue by LezJ** [mengmenm/SMIL#17]
Hi there, thanks for your inspiring work. I am playing with the codes but find that the training can't achieve reported performance in the paper. Information are listed:

- Dataset: avMNIST
- Settings: Upper bound (where both images and audios are always available).
- Implementation: By simply changing the `per_class_num` argument from default 21 - 105.
- Reported performance in the paper: Over 98% acc in the paper.
- My results: Over 96% acc when training acc is 100% and training loss is appoarching 0 (see screenshot below).
![image](https://github.com/user-attachments/assets/04dc4422-c40b-4e16-9b84-5fc64c876f6c)

Do you have any ideas why I can't get the reported results?
Thanks in advance!

**Comment by LezJ** [comment]
Also, the codes to load pretrained unimodal model is commented (see screenshot below) since it is not mentioned in the original paper.
![image](https://github.com/user-attachments/assets/4c5165d9-520b-48ce-9941-854d55d4d89b)

**Comment by LezJ** [comment]
It's finally greater than 98% acc by loading pretrained unimodal model's weights. Is it how you achieved this, or you just train the upper bound model end2end? Thanks.
